specs/Standards/Storage.md

```diff
@@ -77,7 +77,7 @@ Alice registers her account on a fungible token contract.
 
        {
          total: '2350000000000000000000',
-         available: '2350000000000000000000'
+         available: '0'
```

Comment:
This is not the behavior I'm seeing. Just tried it (again) now.

```bash
mike@n  ~/near/near-sdk-rs/examples/fungible-token   hotfix/update-ft ± near delete 141.mike.testnet mike.testnet && near create-account 141.mike.testnet --masterAccount mike.testnet
Deleting account. Account id: 141.mike.testnet, node: https://rpc.testnet.near.org, helper: https://helper.testnet.near.org, beneficiary: mike.testnet
Transaction Id 2JqLijtspS5NxuYrwgq9B9e273XrXCirSYYm7FCaWX1z
To see the transaction in the transaction explorer, please open this url in your browser
https://explorer.testnet.near.org/transactions/2JqLijtspS5NxuYrwgq9B9e273XrXCirSYYm7FCaWX1z
Account 141.mike.testnet for network "default" was deleted.
Saving key to '/Users/mike/.near-credentials/default/141.mike.testnet.json'
Account 141.mike.testnet for network "default" was created.
 mike@n  ~/near/near-sdk-rs/examples/fungible-token   hotfix/update-ft ± near deploy 141.mike.testnet --wasmFile res/fungible_token.wasm --initFunction new --initArgs '{"owner_id": "141.mike.testnet", "total_supply": "1000000", "version": "1.0.0", "name": "Mochi Tokens", "symbol": "MOCHI", "reference": "https://example.com/mochi.json", "reference_hash": "7c879fa7b49901d0ecc6ff5d64d7f673da5e4a5eb52a8d50a214175760d8919a", "decimals": 19}'
Starting deployment. Account id: 141.mike.testnet, node: https://rpc.testnet.near.org, helper: https://helper.testnet.near.org, file: res/fungible_token.wasm
Transaction Id AZ3SxqswjfrPgP8tbr6ijjF1LWtSsYHiUzWtws6beXQs
To see the transaction in the transaction explorer, please open this url in your browser
https://explorer.testnet.near.org/transactions/AZ3SxqswjfrPgP8tbr6ijjF1LWtSsYHiUzWtws6beXQs
Done deploying and initializing 141.mike.testnet
 mike@n  ~/near/near-sdk-rs/examples/fungible-token   hotfix/update-ft ± near call 141.mike.testnet storage_deposit '' --amount '0.00235' --accountId mike.testnet
Scheduling a call: 141.mike.testnet.storage_deposit() with attached 0.00235 NEAR
Transaction Id HPFrXoxtUo8KMKVepnGjYpeU9aiEhwK2zfQaC5wMFNrv
To see the transaction in the transaction explorer, please open this url in your browser
https://explorer.testnet.near.org/transactions/HPFrXoxtUo8KMKVepnGjYpeU9aiEhwK2zfQaC5wMFNrv
{
  total: '2350000000000000000000',
  available: '2350000000000000000000'
}
```

Reply:
Does that indicate an error in the standard-under-draft, or an error in this contract's adherence to it?

Reply:
that is a bug in the contract - `available` balance should be 0. Once the required min storage balance is deposited, it cannot be withdrawn via the account storage API.

Reply:
Then we should introduce an API to withdraw minimum storage balance somehow into the StorageManager standard. Because right now the minimum storage balance is non reclaimable.